src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/PSHCommon/Common.ps1

```diff
@@ -286,68 +357,78 @@ function Wait-Function
     }
 }
 
-
 <#
 .SYNOPSIS
-Waits for specified duration if not-mocked, otherwise skips wait.
+Wait for specified duration if not-mocked, otherwise skips wait.
 
 .PARAMETER timeout
 Timeout in seconds
 #>
 function Wait-Seconds
 {
-    param([int] $timeout)
-    
-    [Microsoft.WindowsAzure.Testing.TestUtilities]::Wait($timeout * 1000)
-}
+    param(
+        [int] $timeout
+    )
 
+    Start-Sleep -Seconds $timeout
```

Comment:
revert this back to [Microsoft.WindowsAzure.Testing.TestUtilities]::Wait. We do not want to do sleep waits in playback mode.